docs/overview/firefly_supernode.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: default
+title: Introduction to Supernodes
+parent: Understanding FireFly
+nav_order: 1
+---
+
+# Introduction to Supernodes
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## What is a Supernode
+
+Over the last decade of enterprise blockchain projects, architects and developers have realized
+that they need much more than a blockchain node for their projects to be successful.
+
+The development stack needed for an enterprise grade Web3 application,
+is just as sophisticated as the stack required for the Web 2.0 applications
+that came before.
+
+A raw blockchain simply node is not enough.
+
+## Your project with or without a Supernode
+
+![Without FireFly / with FireFly](../images/without_firefly_with_firefly.png)
+
+So your choice as a development team for a blockchain project becomes whether you build
+and update all of the "plumbing" / "middleware" components needed underneath your business
+logic yourself, or whether you look for pre-built solutions.
+
+The Hyperledger FireFly approach is to allow the community to collaborate on the development and hardening of
+these component, across industries and projects. Then fit them into an open source, enterprise grade,
+pluggable development and runtime stack... the _Supernode_.
+
+The application then code against these APIs, and can be confident that the business logic that works
+on their local laptop against a sandbox, is being written in a way that scales to an enterprise
+decentralized application and can be deployed against one or more public/private blockchains in production.
+
+Thus allowing development teams to focus on differentiation where it matters - at the solution layer.
+
+## Types of blockchain project that benefit
+
+There are two main reasons your project might be exploring Supernodes, and considering the open source
+approach of Hyperledger FireFly.
+
+### Solution builders
+
+Teams building new decentralized web3 solutions, quickly find they are too complex to run without a
+dedicated deployment stack and private database. Particularly in the enterprise space, due to data
+security, access management privacy concerns.
+
+For the solution to be successful, it needs decentralized deployment to multiple parties. So the
+complexity of a bespoke stack of an individual solution can be a barrier to adoption.
```

Comment:
I get what this is trying to say. Maybe a different word choice for "an individual solution" would make the point more clearly?

Reply:
I did a bunch of tinkering to this section to try and make it clearer